sortdb/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "maintainer": "namrata.deshpande4@gmail.com",
+  "manifest_version": "1.0.0",
+  "name": "sortdb",
+  "short_description": "Datadog support for sortdb monitoring",
+  "guid": "806dcbd7-3686-4472-9435-2049729847c1",
+  "support": "contrib",
+  "supported_os": ["linux","mac_os","windows"],
```

Comment:
Can we add the fields like `create_events` so we can keep this more uniform with integrations-core checks?